pychunkedgraph/graph/analysis/pathing.py

```diff
@@ -70,14 +73,40 @@ def get_children_at_layer(
     return np.concatenate(children_at_layer)
 
 
-def get_lvl2_edge_list(cg, node_id: np.uint64):
+def get_lvl2_edge_list(
+    cg,
+    node_id: np.uint64,
+    bbox: typing.Optional[typing.Sequence[typing.Sequence[int]]] = None,
+):
     """get an edge list of lvl2 ids for a particular node
 
     :param cg: ChunkedGraph object
     :param node_id: np.uint64 that you want the edge list for
+    :param bbox: Optional[Sequence[Sequence[int]]] a bounding box to limit the search
     """
 
-    lvl2_ids = get_children_at_layer(cg, node_id, 2)
+    if bbox is None:
+        # maybe temporary, this was the old implementation
```

Comment:
if we expect this to work perhaps we should just put in a test and make this code simplier.. on the other hand this is guarunteed to not be a breaking change so it's not a bad way to go.

Reply:
elected to keep it this way with the comment for now, but in theory the other function should handle both cases